plugins/commands/advisor/src/main/kotlin/AdvisorCommand.kt

```diff
@@ -125,6 +125,8 @@ class AdvisorCommand : OrtCommand(
         val advisor = Advisor(distinctProviders, ortConfig.advisor)
 
         val ortResultInput = readOrtResult(ortFile)
+
+        @Suppress("ForbiddenMethodCall")
```

Comment:
Please also explain in the commit message in which cases the new rule is suppressed / it is ok to suppress the rule.